chia/full_node/mempool_manager.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, coin_store: CoinStore, consensus_constants: ConsensusConstant
         self.mempool_max_total_cost = int(self.constants.MAX_BLOCK_COST_CLVM * self.constants.MEMPOOL_BLOCK_BUFFER)
 
         # Transactions that were unable to enter mempool, used for retry. (they were invalid)
-        self.potential_cache = PendingTxCache(self.constants.MAX_BLOCK_COST_CLVM * 5)
+        self.potential_cache = PendingTxCache(self.constants.MAX_BLOCK_COST_CLVM * 1)
```

Comment:
Does this make the pending cache smaller?

Reply:
Yes, that cache is used only for items that fail due to time locks. It's intended for 1-off block misses mostly. (doesn't make sense to have anything larger than that)